report/Outline/testPiler.tex

```diff
@@ -0,0 +1,31 @@
+\section{Test Generator Plugin}
+We have chosen to only focus on Flutter development since it allows us to narrow the technical challenges into a known domain for the members of the group.
+We envision a plugin for Flutter that can be installed and will run as part of the test suite in the Flutter application.
+Since the Flutter Framework has a pre-configured test framework we envision our plugin to generate code that is compatible with the build-in test environment of Flutter.
+Thereby leaving next to no setup required.
+
+We envision our plugin to be configured to connect to the task and screen definitions from the application via a shared backend API.
+Our plugin will then be able to parse the data structure used by the Mobile Customer Application to automatically generate executable tests.
+
+We can already now see some big issues with this approach such as, what if the developer has no or unstable internet connection.
+What about implementation details that are not part of the knowledge domain of the Mobile Customer Application.
+
+Therefore we envision a compromise from our initial idea, we will still build a plugin but instead of building a Flutter only plugin, we will build a more general package that can be used either in a Flutter project or in a stand-alone project. 
```

Comment:
```suggestion
Therefore we envision a compromise from our initial idea. 
We will still build a plugin, but instead of building a Flutter only plugin, we will build a more general package that can be used either in a Flutter project or in a stand-alone project. 
```